core/src/main/java/org/apache/lucene/queryparser/classic/MapperQueryParser.java

```diff
@@ -252,7 +253,7 @@ private Query getFieldQuerySingle(String field, String queryText, boolean quoted
                     Query query = null;
                     if (currentFieldType.useTermQueryWithQueryString()) {
                         try {
-                            query = currentFieldType.termQuery(queryText, parseContext);
+                            query = currentFieldType.termQuery(queryText, new QueryGenerationContext(parseContext));
```

Comment:
I wasn't sure about that. If it is temporary we can leave it, let's just make sure we don't forget if we need to take any action in the future